.github/workflows/release-eql.yml

```
@@ -12,6 +12,10 @@ on:
   # Useful for debugging
   workflow_dispatch:
 
+env:
+  FORCE_JAVASCRIPT_ACTIONS_TO_NODE24: true
+  MISE_VERBOSE=1
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)